default.nix

```diff
@@ -207,6 +207,12 @@ let
     name = "hnix";
     root = pkgs.nix-gitignore.gitignoreSource [ ] ./.;
 
+    overrides = self: super: {
+      # 2020-06-26 dontCheck helps us avoid conflicts with neat-interpolation's
+      # test dependencies
+      neat-interpolation = pkgs.haskell.lib.dontCheck super.neat-interpolation_0_5_1;
```

Comment:
Should I comment on why we need the override in the first place?

Reply:
+1 .. as a note for our future self and easier removal when not needed.

Reply:
Done!